### PR TITLE
feat(changelog): transactional-email-added-tem 2023-01-19

### PR DESCRIPTION
**Pull request by ofranc**
Reporter: Chloe PANG

This PR is a new changelog contribution. It has been created automatically.

Make sure to review it carefully before merging it.

### CODE REVIEW

File: changelog/january2023/2023-01-19-transactional-email-added-tem.mdx [hunk 1]
```diff
@@ -0,0 +1,16 @@
+---
+title: TEM
+status: added
+author:
+    fullname: 'Join the #[transactional-email] channel on Slack.'
```
Comment (bene2k1) [inline changelog/january2023/2023-01-19-transactional-email-added-tem.mdx:5]:
```suggestion
    fullname: 'Join the #transactional-email channel on Slack.'
```

File: changelog/january2023/2023-01-19-transactional-email-added-tem.mdx [hunk 2]
```diff
@@ -0,0 +1,16 @@
+---
+title: TEM
```
Comment (nedaehm) [inline changelog/january2023/2023-01-19-transactional-email-added-tem.mdx:2]:
```suggestion
title: The API documentation for Transactional Email is now live!
```

File: changelog/january2023/2023-01-19-transactional-email-added-tem.mdx [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+status: added
+author:
+    fullname: 'Join the #[transactional-email] channel on Slack.'
+    url: 'https://scaleway-community.slack.com/archives/C0358JE0XEK'
+date: 2023-01-19
+category: managed-services
+product: transactional-email
+---
+
+- The API documentation for TEM is now live! Check it out here: https://developers.scaleway.com/en/products/transactional_email/api/
```
Comment (nedaehm) [inline changelog/january2023/2023-01-19-transactional-email-added-tem.mdx:12]:
```suggestion
- The API documentation for Transactional Email is now live. [Check it out!] (https://developers.scaleway.com/en/products/transactional_email/api/)
```

File: changelog/january2023/2023-01-19-transactional-email-added-tem.mdx [hunk 4]
```diff
@@ -0,0 +1,16 @@
...
+
+- TEM now supports the following attachment types: calendar format (text/ics), (application/ics) and csv files.
```
Comment (nedaehm) [inline changelog/january2023/2023-01-19-transactional-email-added-tem.mdx:14]:
```suggestion
```

Comment (nedaehm) [reply changelog/january2023/2023-01-19-transactional-email-added-tem.mdx:14]: I think this should be another separate entry

File: changelog/january2023/2023-01-19-transactional-email-added-tem.mdx [hunk 5]
```diff
@@ -0,0 +1,16 @@
...
+    fullname: 'Join the #transactional-email channel on Slack.'
+    url: 'https://scaleway-community.slack.com/archives/C0358JE0XEK'
+date: 2023-01-19
+category: managed-services
+product: transactional-email
+---
+
+- The API documentation for Transactional Email is now live. [Check it out!] (https://developers.scaleway.com/en/products/transactional_email/api/)
+
+- TEM now supports the following attachment types: calendar format (text/ics), (application/ics) and csv files.
```
Comment (nedaehm) [inline changelog/january2023/2023-01-19-transactional-email-added-tem.mdx:14]:
```suggestion
```

File: changelog/january2023/2023-01-19-transactional-email-added-tem.mdx [hunk 6]
```diff
@@ -0,0 +1,16 @@
+---
+title: The API documentation for Transactional Email is now live!
+status: added
+author:
+    fullname: 'Join the #transactional-email channel on Slack.'
+    url: 'https://scaleway-community.slack.com/archives/C0358JE0XEK'
```
Comment (bene2k1) [inline changelog/january2023/2023-01-19-transactional-email-added-tem.mdx:6]:
```suggestion
    url: 'https://slack.scaleway.com/'
```

**Review by bene2k1 — APPROVED** [review]

File: changelog/january2023/2023-01-19-transactional-email-added-tem-2.mdx [hunk 7]
```diff
@@ -0,0 +1,15 @@
...
+status: added
+author:
+    fullname: 'Join the #transactional-email channel on Slack.'
+    url: 'https://slack.scaleway.com/'
+date: 2023-01-19
+category: managed-services
+product: transactional-email
+---
+
+- The API documentation for Transactional Email is now live. [Check it out!] (https://developers.scaleway.com/en/products/transactional_email/api/)
```
Comment (bene2k1) [inline changelog/january2023/2023-01-19-transactional-email-added-tem-2.mdx:12]:
```suggestion
- The API documentation for Transactional Email is now live. [Check it out!](https://developers.scaleway.com/en/products/transactional_email/api/)
```

**Pull request merged**